targets/x86-geode

```diff
@@ -18,5 +18,9 @@ packages {
 	'kmod-via-velocity',
 }
 
-factory_image('x86-geode', 'combined-squashfs', '.img.gz')
-sysupgrade_image('x86-geode', 'combined-squashfs', '.img.gz')
+device('x86-geode', 'generic', {
+	factory = '-squashfs-combined',
+	factory_ext = '.img.gz',
```

Comment:
What about VMDK/VDI images?

Reply:
For geode?

Reply:
Oops, I meant to add this to `targets/x64.inc`.